### ISSUE TITLE
Upcoming deactivation from sensoApp, multiMATIC

### ISSUE THREAD

**Issue by Presl1983** [thomasgermain/vaillant-component#260]
I think this is bad for us - correct?

Mit 31.03.2024 werden alle aktuellen Vaillant-Apps (sensoApp, multiMATIC) deaktiviert. Nach diesem Datum ist keine Migration auf die myVaillant-App mehr möglich. Auch die verknüpfte Internet-Kommunikations-Box (VR-9XX) Ihres Vaillant-Produktes kann dann nicht mehr auf die aktuellste Firmware upgedatet werden und verliert ihre Funktion. Sollten Sie nicht bis zum 31.03.2024 umsteigen und ab dem 01.04.2024 ihre Anlage wieder mit dem Internet verbinden wollen, ist eine kostenpflichtige Neuschaffung einer aktuellen Internet-Kommunikations-Box (VR-940f) notwendig.

**Comment by thomasgermain** [comment]
Any source ?

It's not bad as long as they offer the possibility to migrate before the deadline

**Comment by Presl1983** [comment]
https://www.vaillant.at/privatanwender/produkte/mobile-apps/myvaillant-app/

![image](https://github.com/thomasgermain/vaillant-component/assets/142180290/ffa6f903-90ac-426e-bff1-ace03b1b65a3)

**Comment by pepsonEL** [comment]
I have gateway VR920 and Multimatic VRC700/6  with app Multimatic and still no get upodate to myVaillant. When i contact with Poland suppoert Vaillant , he tell me that Multimatic should be upgrade also to end 31.03.2024

**Comment by pepsonEL** [comment]
Yes... i also have the same info.... But first info about update was that should be to 31.12.2023.... now the date was move to 31.03.2024..... ehhhhh Vaillant. In my opinion Multimatic works very very bad.... big loags ans other problems.,

**Comment by Presl1983** [comment]
Are you able to get data from your System at the Moment via the Smartphone App? My System is offline... Support dont know why😔

**Comment by pepsonEL** [comment]
Yes now i have access to get info... But from yesterday i also dont have and have offline,. Today i power off and nthen power on my gateway and works ok.

**Comment by Presl1983** [comment]
Thx m8. i made a Reboot again today and now it works...

**Comment by Presl1983** [comment]
^^ i know them... lets hope the integration will work after 31.3 :/

**Comment by pepsonEL** [comment]
I belive that before :)

**Comment by ttuuxxeerr** [comment]
That will piss me a lot. I just installed a new system and the technician suggested me to use the module Internet VR921. I hope this one will become compatible with the new app.

**Comment by zolakt** [comment]
Does switching to the new app affect this integration in any way?

**Comment by ttuuxxeerr** [comment]
> Does switching to the new app affect this integration in any way?
> 
> Btw, even though their wizard clams so, it didn't migrate my existing account. I had to create a new one in the new app. So I guess, I at least need to remove and readd the integration with the new credentials?

did you receive the invitation to migrate? I did not receive anything yet.

**Comment by zolakt** [comment]
> > Does switching to the new app affect this integration in any way?
> > Btw, even though their wizard clams so, it didn't migrate my existing account. I had to create a new one in the new app. So I guess, I at least need to remove and readd the integration with the new credentials?
> 
> did you receive the invitation to migrate? I did not receive anything yet.

Yeah, I did, but declined it initially, since I was afraid it will mess up this integration.

Now I have a permanent warning in the sensoApp telling me to migrate.
It brings up this wizard, which says it will create a new account... but it doesn't.

First time I clicked "Start update" it just goes to the play store and downloads the new app.
Every subsequent time I click it, it does nothing, flashes something for a second and closes it immediately.

I can create a new account in the new app, and add the device there, with the serial number. But I'm not sure if that is the proper way to do it. And I'd still like to know what will happen with this integration if I switch?
![image](https://github.com/thomasgermain/vaillant-component/assets/1048531/3558f262-ae5f-45e0-9a10-5ae11f33b34c)

**Comment by ttuuxxeerr** [comment]
> > > Does switching to the new app affect this integration in any way?
> > > Btw, even though their wizard clams so, it didn't migrate my existing account. I had to create a new one in the new app. So I guess, I at least need to remove and readd the integration with the new credentials?
> > 
> > 
> > did you receive the invitation to migrate? I did not receive anything yet.
> 
> Yeah, I did, but declined it initially, since I was afraid it will mess up this integration.
> 
> Now I have a permanent warning in the sensoApp telling me to migrate. It brings up this wizard, which says it will create a new account... but it doesn't.
> 
> First time I clicked "Start update" it just goes to the play store and downloads the new app. Every subsequent time I click it, it does nothing, flashes something for a second and closes it immediately.
> 
> I can create a new account in the new app, and add the device there, with the serial number. But I'm not sure if that is the proper way to do it. And I'd still like to know what will happen with this integration if I switch? ![image](https://private-user-images.githubusercontent.com/1048531/294163720-3558f262-ae5f-45e0-9a10-5ae11f33b34c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQzNjczMzksIm5iZiI6MTcwNDM2NzAzOSwicGF0aCI6Ii8xMDQ4NTMxLzI5NDE2MzcyMC0zNTU4ZjI2Mi1hZTVmLTQ1ZTAtOWExMC01YWUxMWYzM2IzNGMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDEwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMDRUMTExNzE5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZGY5NWE1MzdkOGFiYjQ3MzU1NjE1NDUxMGY5Mzg1ZTM0NWVhYzY1MTVkNzQzZTYzN2Y4ZjczMGFiMTI1YmQ3YiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.FHIj89k721PCOH33zXMDr_T9GB1HnKFK2qR04CtZNVY)

I believe I don't receive the invitation because I am using ambisense and it looks it is not yet supported.  Maybe if you migrate you need to use a different plugin in HA:

https://github.com/signalkraft/mypyllant-component

**Comment by ttuuxxeerr** [comment]
Btw, answer from Vaillant about my question on this subject:

Good morning,

Please note that all our customers will have to migrate to the new myVAILLANT application.
Which means that you too will have to migrate from old (slower) servers to new (faster) servers.
The migration should be completed by the end of January, unless the process is delayed.
You will receive, automatically in the current application, a pop-up message asking you to migrate.


Sincerely,
Technical Helpdesk,
Valiant

**Comment by zolakt** [comment]
> Btw, answer from Vaillant about my question on this subject:

How did you contact them?
I've sent an email to app-support@vaillant-group.com, but got no reply.

**Comment by ttuuxxeerr** [comment]
> > Btw, answer from Vaillant about my question on this subject:
> 
> How did you contact them? I've sent an email to [app-support@vaillant-group.com](mailto:app-support@vaillant-group.com), but got no reply.

using their contact form: https://www.vaillant.be/particuliers/contact/question-technique/

**Comment by iamgeert** [comment]
> Btw, answer from Vaillant about my question on this subject:
> 
> Good morning,
> 
> Please note that all our customers will have to migrate to the new myVAILLANT application. Which means that you too will have to migrate from old (slower) servers to new (faster) servers. The migration should be completed by the end of January, unless the process is delayed. You will receive, automatically in the current application, a pop-up message asking you to migrate.
> 
> Sincerely, Technical Helpdesk, Valiant

Did you ask about Ambisense?

**Comment by ttuuxxeerr** [comment]
> > Btw, answer from Vaillant about my question on this subject:
> > Good morning,
> > Please note that all our customers will have to migrate to the new myVAILLANT application. Which means that you too will have to migrate from old (slower) servers to new (faster) servers. The migration should be completed by the end of January, unless the process is delayed. You will receive, automatically in the current application, a pop-up message asking you to migrate.
> > Sincerely, Technical Helpdesk, Valiant
> 
> Did you ask about Ambisense?

Yes, waiting their reply

**Comment by zolakt** [comment]
FYI: I got a response from Vaillant on how to migrate.
The instructions in the app are wrong (or at least I misinterpreted them). There is no automatic account creation.

Here are the instructions:
0. There will be an email and App notification when your system is ready to be migrated (I got those already)
1. Create a new Account in the myVAILLANT app
2. Login to the myVAILLANT APP
3. SET up Home (name the gateway and add the gateway serial number
4. Input the USERNAME and PASSWORD from the OLD SensoAPP account (so the new server can access the old account to transfer the data)
5. After this the system and gateway firmware will update for 5 to 20 minutes to make the account and system compatible for the new system

**Comment by ttuuxxeerr** [comment]
> > > Btw, answer from Vaillant about my question on this subject:
> > > Good morning,
> > > Please note that all our customers will have to migrate to the new myVAILLANT application. Which means that you too will have to migrate from old (slower) servers to new (faster) servers. The migration should be completed by the end of January, unless the process is delayed. You will receive, automatically in the current application, a pop-up message asking you to migrate.
> > > Sincerely, Technical Helpdesk, Valiant
> > 
> > 
> > Did you ask about Ambisense?
> 
> Yes, waiting their reply

Original answer:

je tiens a vous rassurer tout de suite, ca sera compatible avec une passerelle internet qui va devoir migrer vers la nouvelle application.
Si vous acheter une passerelle fonctionnant sur la nouvelle application SANS devoir migrer, ......la ca ne fonctionnera pas.

Les installations ambisense seront les dernières a migrer.

Google translated: 

I want to reassure you straight away, it will be compatible with an internet gateway which will have to migrate to the new application.
If you buy a gateway running on the new application WITHOUT having to migrate,......it will not work.

Ambisense installations will be the last to migrate.

**Comment by iamgeert** [comment]
I got this response for the Ambisense support from Vaillant 

> Het klopt dat er voor 31/3/2024 gemigreerd zal worden naar de MyVaillant app.
> 
> Er was voorzien dat de Ambisense vanaf eind vorig jaar zichtbaar gingen zijn in de MyVaillant app.
> Maar door een aantal problemen is dit nu voorzien voor ten laatste eind februari.
> 
> Een exacte datum hebben we momenteel nog niet doorgekregen. 

_Google translated:_

> It is true that migration to the MyVaillant app will take place before 31/3/2024.
> 
> It was planned that the Ambisense would be visible in the MyVaillant app from the end of last year.
> But due to a number of problems, this is now scheduled for the end of February at the latest.
> 
> We have not yet received an exact date.

**Comment by zolakt** [comment]
I've successfully migrated to the MyVaillant app.
The SensoApp stops working immediatelly, therefore this integration as well.
I've switched to https://github.com/signalkraft/mypyllant-component, which seems to work fine. Has even more sensors and controls.

However, I do not have any consumption data now!? In the myVaillant app or the mypyllant integration.
I see screenshots online that the myVaillant app should have this, but all that I see on that screen are water pressure and flow temperatures. Did anyone have the same issue? Do I need to do something do enable it, or is this Vaillant punishing migrating customers for some reason?
![Screenshot (16 Jan 2024 12 13 49)](https://github.com/thomasgermain/vaillant-component/assets/1048531/a53f7884-5a6c-45c2-8446-072e34077f31)

**Comment by Tharanis** [comment]
I switched yesterday. So i could tell something about the problems after the switch-Process.

My Components are: Vaillant VC 15 CS/1-7 ecoTEC exclusive and Vaillant sensoNET VR 921. Until yesterday i´d used this integration. In the Switch-Progress i´d click the update-Button and i think my VR921 got a new software-version only compatible with myvaillant. So... since yesterday i only can use the myvaillant-App. This Integration und the myphyllant-Integration for homeassistant are both not working for me. This integration don´t show my Vaillant-Components anymore and myphyllant don´t show them too. Perhaps a Problem so short after the switch and vaillant has to make some clicks. 

I think, i had to the switch to myvaillant... but i´m not happy that my ha-automations don´t work anymore. I hope the best for the future...

**Comment by pepsonEL** [comment]
I am still waiting for this news from Vaillant. What is a topis this email which you recived ?

**Comment by vozda** [comment]
> I am still waiting for this news from Vaillant. What is a topis this email which you recived ?

First e-mail contained information that migration will be soon, and second e-mail next day announcing that migration was available + notification in the Multimatic app.

**Comment by vozda** [comment]
> FYI: I got a response from Vaillant on how to migrate. The instructions in the app are wrong (or at least I misinterpreted them). There is no automatic account creation.
> 
> Here are the instructions:
> 
> 1. There will be an email and App notification when your system is ready to be migrated (I got those already)
> 2. Create a new Account in the myVAILLANT app
> 3. Login to the myVAILLANT APP
> 4. SET up Home (name the gateway and add the gateway serial number
> 5. Input the USERNAME and PASSWORD from the OLD SensoAPP account (so the new server can access the old account to transfer the data)
> 6. After this the system and gateway firmware will update for 5 to 20 minutes to make the account and system compatible for the new system
> 
> Btw @thomasgermain do you have any intentions of updating this integration to the new api, or should we all just switch to https://github.com/signalkraft/mypyllant-component? As I've mentioned in another thread, I'm reluctant to do radical changes like these in the middle of the heating season. Really poor timing by Vaillant

myPyllant does not currently support Multimatic 700.

**Comment by zolakt** [comment]
> > FYI: I got a response from Vaillant on how to migrate. The instructions in the app are wrong (or at least I misinterpreted them). There is no automatic account creation.
> > Here are the instructions:
> > 
> > 1. There will be an email and App notification when your system is ready to be migrated (I got those already)
> > 2. Create a new Account in the myVAILLANT app
> > 3. Login to the myVAILLANT APP
> > 4. SET up Home (name the gateway and add the gateway serial number
> > 5. Input the USERNAME and PASSWORD from the OLD SensoAPP account (so the new server can access the old account to transfer the data)
> > 6. After this the system and gateway firmware will update for 5 to 20 minutes to make the account and system compatible for the new system
> > 
> > Btw @thomasgermain do you have any intentions of updating this integration to the new api, or should we all just switch to https://github.com/signalkraft/mypyllant-component? As I've mentioned in another thread, I'm reluctant to do radical changes like these in the middle of the heating season. Really poor timing by Vaillant
> 
> myPyllant does not currently support Multimatic 700.

Could be.

I've made the switch to the new app and myPlyiant, without any issues, with this setup:
- boiler ecoTEC plus 30/36CS
- main circuit thermostat sensoComfort VRC 720
- secondary circuit thermostat VR 92
- sensoNET VR 921 module

Everything works fine, had no issues. 2 heating circuits and dhw.
All the controls and sensors are working immidiately.
The consumptions sensors take some time to start pulling data. So either wait a little between switching to the new app (until it populates consumption data) and myPyllant. Or reload myPyllant if you set it up before the data is populated.

**Comment by camhef** [comment]
I contacted Vaillant last week by email and this is their response:

> The sensoAPP and 700 apps are in the process of being upgraded to the myVAILLANT app on a new server. 

> The migration to the myVAILLANT app currently underway in batches and will be phased in by invitation only.

> The app migration should be complete by early 2024, basic system setups first and complex systems (including ambiSENSE and 700 systems from late January to March 24).

As I have ambiSENSE valves I will probably have to wait until March 2024.

**Comment by GLehnhoff** [comment]
### The above discussion is not a Home Assistant discussion
Maybe I am wrong. However, is this the right place to discuss the Vaillant App upgrade? I don't think so. This is a Home Assistant bug report place. Saying that, here is my problem:

1. I upgraded from multiMATIC app to myVAILLANT app.
2. The Home Assistant Multimatic integration seems not to work anymore.
